lib/perl/Genome/WorkflowBuilder/DAG.pm

```diff
@@ -210,9 +211,10 @@ sub get_ptero_builder_task {
 sub get_ptero_builder_for_process {
     require Ptero::Builder::Workflow;
 
-    my ($self, $process_id) = Params::Validate::validate_pos(@_, 1, 1);
+    my ($self, $process) = Params::Validate::validate_pos(@_, 1, 1);
+    my $process_id = $process->id;
```

Comment:
It does not appear that this `$process_id` is used any longer, so line 215 could be deleted.